packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -105,7 +109,14 @@ export const AuthModal = ({
 
         const handlePasswordlessLogin = async (email) => {
             try {
-                await authorizePasswordlessLogin.mutateAsync({userid: email})
+                const callbackURL = isAbsoluteURL(passwordlessConfigCallback)
+                    ? passwordlessConfigCallback
+                    : `${appOrigin}${passwordlessConfigCallback}`
```

Comment:
Would we consider creating the callbackURL outside of `handlePasswordlessLogin` so this logic isn't run everytime a user clicks passwordless login?